packages/web-app-files/src/views/FilesDrop.vue

```diff
@@ -89,6 +91,9 @@ export default defineComponent({
     const { getInternalSpace } = useGetMatchingSpace()
     useUpload({ uppyService })
 
+    const { currentTheme } = storeToRefs(themeStore)
+    const themeSlogan = computed(() => currentTheme.value.common.slogan)
```

Comment:
Does this need to be computed, as storeToRefs builds already a reactive object, or does this not apply to nested objects?

Reply:
As far as I understand it: It's not necessary in this case. `currentTheme` is already a `Ref` because of `storeToRefs`.  Meaning the consuming code could go directly via `unref(currentTheme).common.slogan`.

It doesn't hurt to have a computed here though, I guess it's used as a "shortcut".